src/cairoWriter/writers/literalWriter.ts

```diff
@@ -31,16 +29,12 @@ export class LiteralWriter extends CairoASTNodeWriter {
         return [`0x${node.hexValue}`];
       }
       case LiteralKind.HexString:
-        switch (primitiveTypeToCairo(node.typeString)) {
-          case 'Uint256': {
-            return [
-              `Uint256(low=0x${node.hexValue.slice(32, 64)}, high=0x${node.hexValue.slice(0, 32)})`,
-            ];
-          }
-          case 'felt':
-            return [`0x${node.hexValue}`];
-          default:
-            throw new TranspileFailedError('Attempted to write unexpected cairo type');
+        if (primitiveTypeToCairo(node.typeString) as CairoPrimitiveIntType) {
```

Comment:
This condition is always going to be truth, I guess you want to compare the string?
```suggestion
        if (primitiveTypeToCairo(node.typeString) === "smth?") {
```
Same thing happens in line 10